test/unit/signup/account_manager_test.rb

```diff
@@ -291,7 +291,7 @@ class PlansWithDefaultsTest < Signup::AccountManagerTest
 
       should 'yield errors to proc' do
         errors = []
-        @plans.error_proc = proc { |error| errors << error }
```

Comment:
because this was the only reason to have this:
https://github.com/3scale/porta/blob/f5bdc1a124993080add3d00099b4c7e27e438055/lib/signup/account_manager.rb#L84

as you can see, it is not called from any other class other than itself:
![image](https://user-images.githubusercontent.com/11318903/69243990-72653980-0ba4-11ea-8418-c1a588fd8ca2.png)